packages/perseus/src/widgets/numeric-input/numeric-input.test.ts

```diff
@@ -93,9 +93,6 @@ describe("static function getOneCorrectAnswerFromRubric", () => {
         const singleAnswer =
             NumericInputWidgetExport.getOneCorrectAnswerFromRubric?.({
                 answers,
-                labelText: "",
-                size: "medium",
-                static: false,
```

Comment:
`NumericInputWidgetExport.getOneCorrectAnswerFromRubric` only takes `answers` and `coefficient` props in the args object.